fem/restriction.hpp

```diff
@@ -221,7 +221,7 @@ class FaceRestriction : public Operator
 /// Operator that extracts Face degrees of freedom for H1 FiniteElementSpaces.
```

Comment:
This old comment should be updated, since it is no longer just H1.

Reply:
Fixed in 8869ad64a